libs/utils/utils.js

```diff
@@ -144,7 +144,7 @@ export const MILO_EVENTS = { DEFERRED: 'milo:deferred' };
 const LANGSTORE = 'langstore';
 const PREVIEW = 'target-preview';
 const PAGE_URL = new URL(window.location.href);
-const SLD = PAGE_URL.hostname.includes('.aem.') ? 'aem' : 'hlx';
+export const SLD = PAGE_URL.hostname.includes('.aem.') ? 'aem' : 'hlx';
```

Comment:
I know that `SLD` was already in use but SLD is kind of cryptic - maybe a more descriptive name?  `AEM_DOMAIN`?

Reply:
"second level domain" 😁 (If I remember correct) I know @overmyheadandbody and myself had some discussions back when we introduced it and couldn't settle on anything great.

`AEM_DOMAIN` sounds a bit like it's a fully qualified domain 🤔 Theres quite a few references, I'm fine to change it - but we should settle on something on agreement.

I'm still fine with taking `AEM_DOMAIN` @overmyheadandbody opinion?

Reply:
Yeah not sure that `AEM_DOMAIN` is best, but I had to look up SLD acronym to figure out what that represented.  I think even `AEM_SLD` would be better.  

I guess with `.hlx.` being deprecated and [eventually end-of-lifed](https://www.aem.live/docs/deprecation#hlxlive-and-hlxpage), this is a temp const that will go away in the future so maybe it doesn't really matter.